app/models/authentication_method/serializer.rb

```diff
@@ -0,0 +1,37 @@
+# @todo Maybe we want to try?
+# http://jsonapi-rb.org/guides/serialization/defining.html
```

Comment:
I think we 💯 want to follow the JSONAPI spec if/when we are going to have a Proper API. I don't want to gate this PR on adjusting to the JSONAPI spec, though, especially since for now this is an affordance for our tests.

But if y'all feel moved to include JSONAPI in this PR, then 👏🏼 👏🏼 .

Reply:
Yea, I think that's too much for this PR as well, as we want to get it merged in the next 30m